### PR TITLE
Vote-841 -> links migration

### PR DESCRIPTION
**Pull request by clmedders**
**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
https://bixal-projects.atlassian.net/browse/VOTE-841

## Description (optional)
This PR will add link validator test to the project.  Right now these are place holder until content is present on site and will be updated to cover and test the correct parts of the site. 

### Testing steps
1. pull down branch 
2. `lando start`
3. cd into `testing` folder 
4. `npm run cy:open` and check excluded, external and internal links test are passing 
5. `npm run cy:links` -> internal links test headless and `npm run cy:proofer` -> external links test headless

### CODE REVIEW

File: testing/cypress/e2e/excludedLinks/excluded-links.cy.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/// <reference types="Cypress" />
+
+// this test will run out of the pipeline and will be a manaul test each sprint 
+
+const english = require("../../fixtures/sample-links-page.json");
```
Comment (clmedders) [inline testing/cypress/e2e/excludedLinks/excluded-links.cy.js:5]: currently this is just a test json file to make sure the test is running correctly.

File: testing/cypress/e2e/externalLinks/english-external-links-validator.cy.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
+/// <reference types="Cypress" />
+
+
+const allPages = require("../../fixtures/testing-page.json");
```
Comment (clmedders) [inline testing/cypress/e2e/externalLinks/english-external-links-validator.cy.js:4]: same here

File: testing/cypress/e2e/externalLinks/english-external-links-validator.cy.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+              retries: {
+                runMode: 2,
+              },
+            }
+          : {},
+      () => {
+        cy.visit({
+          url: page.route,
+        });
+          // the follow code will be brought back in once content is on site
```
Comment (clmedders) [inline testing/cypress/e2e/externalLinks/english-external-links-validator.cy.js:50]: right now this is just running to make sure everything is functioning... once we get content on the site il bring all this back

File: testing/cypress/e2e/internalLinks/english-internal-links-validator.cy.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+              retries: {
+                runMode: 2,
+              },
+            }
+          : {},
+      () => {
+        cy.visit({
+          url: page.route,
+        });
+          // the follow code will be brought back in once content is on site
```
Comment (clmedders) [inline testing/cypress/e2e/internalLinks/english-internal-links-validator.cy.js:50]: right now this is just running to make sure everything is functioning... once we get content on the site il bring all this back

File: testing/cypress/fixtures/site-pages.json [hunk 5]
```diff
@@ -0,0 +1,287 @@
+[
```
Comment (clmedders) [inline testing/cypress/fixtures/site-pages.json:1]: just bringing this over to be used later on

File: testing/cypress/fixtures/spanish-site.json [hunk 6]
```diff
@@ -0,0 +1,287 @@
+[
```
Comment (clmedders) [inline testing/cypress/fixtures/spanish-site.json:1]: same for the Spanish path

**Comment by clmedders** [comment]
@rayestrada  let me know if you want me to take anything out of this or update anything!

**Review by rayestrada — APPROVED** [review]

**Pull request merged**